sound/soc/sof/ops.h

```diff
@@ -110,6 +110,13 @@ static inline int snd_sof_dsp_parse_platform_ext_manifest(struct snd_sof_dev *sd
 	return 0;
 }
 
```

Comment:
We don't need a new op for this right? We have core_power_up op and you can pass the core is of the last core